time.c

```diff
@@ -34,6 +34,7 @@
 #include "id.h"
 #include "internal.h"
 #include "internal/array.h"
+#include "internal/hash.h"
```

Comment:
@nobu knows more than I do about this, but I think this changes the dependencies, and I don't see an update to the dependency information.

Reply:
Can you please elaborate? I think I am missing something big, where's the dependency information that needs to be updated? In Makefiles?..

Reply:
The dependencies are stored at the bottom of `common.mk`.  They are autogenerated, I think by `tool/update-deps`.

Reply:
Looking at `common.mk`, it definitely needs to be updated.  You can take the line you need to add from the failing CI job: https://github.com/ruby/ruby/actions/runs/3456234830/jobs/5768826516#step:11:13